integrationTests/vm/txsFee/relayedESDT_test.go

```diff
@@ -0,0 +1,118 @@
+package txsFee
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/vmcommon"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/txsFee/utils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRelayedESDTTransferShouldWork(t *testing.T) {
+	testContext := vm.CreatePreparedTxProcessorWithVMs(t, vm.ArgEnableEpoch{})
+	defer testContext.Close()
+
+	relayerAddr := []byte("12345678901234567890123456789033")
+	sndAddr := []byte("12345678901234567890123456789012")
+	rcvAddr := []byte("12345678901234567890123456789022")
+
+	relayerBalance := big.NewInt(10000000)
+	esdtBalance := big.NewInt(100000000)
+	token := []byte("miiutoken")
+	utils.CreateAccountWithESDTBalance(t, testContext.Accounts, sndAddr, big.NewInt(0), token, esdtBalance)
+	_, _ = vm.CreateAccount(testContext.Accounts, relayerAddr, 0, relayerBalance)
+
+	gasPrice := uint64(10)
+	gasLimit := uint64(40)
+	innerTx := utils.CreateESDTTransferTx(0, sndAddr, rcvAddr, token, big.NewInt(100), gasPrice, gasLimit)
+
+	rtxData := utils.PrepareRelayerTxData(innerTx)
+	rTxGasLimit := 1 + gasLimit + uint64(len(rtxData))
+	rtx := vm.CreateTransaction(0, innerTx.Value, relayerAddr, sndAddr, gasPrice, rTxGasLimit, rtxData)
+
+	retCode, err := testContext.TxProcessor.ProcessTransaction(rtx)
+	require.Equal(t, vmcommon.Ok, retCode)
+	require.Nil(t, err)
+
+	_, err = testContext.Accounts.Commit()
+	require.Nil(t, err)
+
+	expectedBalanceSnd := big.NewInt(99999900)
```

Comment:
replace hardcoded with the difference between `esdtBalance` and the value of the transfer

Reply:
I prefer the hardcoded values for tests, also everywhere in these tests I use that pattern.